packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -140,15 +140,31 @@ export interface LoggingConfiguration {
  * One or the other must be passed, and it is invalid to pass both in the same SourceConfiguration.
  */
 export interface SourceConfiguration {
+  /**
+   * The number of times that CloudFront attempts to connect to the origin.
+   * You can specify 1, 2, or 3 as the number of attempts.
```

Comment:
I imagine quite a few users will be hitting this limitation since its fairly un-intuitive. Lets add a validation?